### DISCUSSION TITLE
How to add alt name

### DISCUSSION THREAD

**Discussion by HamasakiBrain** [spatie/laravel-medialibrary#2976] [Q&A] [answered]
**Hello, How to add alt name for image while saving ?**

**Comment by erikn69** [comment, marked as answer]
On name
>  `alt="{{ $media->name }}"`
https://github.com/spatie/laravel-medialibrary/blob/9c3b785f55846fd3cf848c91affe0b35fa383844/resources/views/image.blade.php#L1

Also using custom properties and custom views
https://spatie.be/docs/laravel-medialibrary/v10/advanced-usage/using-custom-properties
https://spatie.be/docs/laravel-medialibrary/v10/responsive-images/customizing-the-rendered-html

**Reply by fredsal** [reply]
Mark this as answered
